packages/schemas/features/blocks/bubbles/audio/schema.ts

```diff
@@ -5,6 +5,7 @@ import { BubbleBlockType } from '../constants'
 export const audioBubbleContentSchema = z.object({
   url: z.string().optional(),
   isAutoplayEnabled: z.boolean().optional(),
+  hideControls: z.boolean().default(false).optional(),
```

Comment:
I'd prefer a field named `areControlsDisplayed` to stay consistent with the repo boolean var convention